### ISSUE TITLE
Keyboard Shortcut

### ISSUE THREAD

**Issue by cyrill62** [jmfayard/tabtab-search#6]
Is it possible to add a keyboard shortcut which will go to address bar & type `tab` ?

**Comment by ArgonQQ** [comment]
That would be amazing!

**Comment by jmfayard** [comment]
@cyrill62 @ArgonQQ 
Question: are you already familiar the keyboard shortcut `⌘+L` (Mac) or `Ctrl+L` or `Alt+D` (Linux/Windows) to go to the Location Bar without leaving the keyboard?

**Comment by ArgonQQ** [comment]
Yes I am. But I still have to type "tab <tab>". Is it possible to create a shortcut which skips the tab <tab> step? Or is it too much effort?

**Comment by jmfayard** [comment]
I will look into it. 
By the way, I noticed that chrome now show by default an icon for all extensions, until the user explicitly hide it.
So there is this a new UX, especially for new users, and we should think about how to use it. 
Teaching about the new keyboard shortcut is a good candidate of what it can do

**Comment by erichiller** [comment]
Being unfamiliar with the chrome extension API myself and what is possible, it would be quite nice to have a hotkey if possible - I gave it a +1.

**Comment by maximelebreton** [comment]
In manifest.json:
```
...
"commands": {
  "NAME_OF_THE_FUNCTION": {
    "suggested_key": { "default": "Alt+Shift+T", "mac": "Command+Shift+T" },
      "description": "TabTab Search"
    }
  }
}
...
```

But I don't know if it's possible to call the Omnibox function?
like the `Ctrl+E` for default search.

Any ideas?
